controllers/chat/group/removeGroupMember.js

```diff
@@ -39,6 +39,8 @@ const removeGroupMember = async (req, res) => {
         text: textDefaultUser,
         own_text: textOwnUser,
         other_text: textTargetUser,
+        other_system_user: userId,
+        better_type: 'remove_member_from_group',
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of `other_system_user` and `better_type` properties seems fine. However, it's important to ensure that the `userId` variable is properly validated and sanitized before being used here to prevent potential security issues such as SQL injection or cross-site scripting (XSS). Also, make sure that the string 'remove_member_from_group' for `better_type` is consistent across your codebase.

```diff
+        other_system_user: sanitize(userId),
+        better_type: 'remove_member_from_group',
```
Please replace `sanitize()` with the actual function you use in your project to sanitize input data.


<!-- This is an auto-generated comment by OSS CodeRabbit -->